docs/usage.md

```diff
@@ -121,6 +121,7 @@ centrifuge,db1,,/<path>/<to>/centrifuge/minigut_cf.tar.gz
 metaphlan,db1,,/<path>/<to>/metaphlan/metaphlan_database/
 motus,db_mOTU,,/<path>/<to>/motus/motus_database/
 ganon,db1,,/<path>/<to>/ganon/test-db-ganon.tar.gz
+kmcp,db1,,/<path>/<to>/ganon/test-db-kmcp.tar.gz
```

Comment:
Add example of passing parameters to KMCP profile (like bracken)

Please also update paragraph 127 as this will also apply there for KCMP

Reply:
Done